.docker/docker-compose.db.yaml

```diff
@@ -22,5 +22,22 @@ services:
     depends_on:
       - vsebude_db
 
+  # for testing availability of redis
+  # docker exec -it vsebude_cache sh
+  # redis-cli
+  # auth redis_admin redis_password_777
+  # if there is no error - everything is OK
+  vsebude_cache:
+    container_name: vsebude_cache
+    build:
+      dockerfile: ./config/redis.Dockerfile
```

Comment:
Instead of separate dockerfile that just copy config use volumes (and command if needed) https://stackoverflow.com/questions/30547274/redis-in-cocker-compose-any-way-to-specify-a-redis-conf-file